lib/core/file.nit

```diff
@@ -663,6 +663,13 @@ class Path
 		return res
 	end
 
+	# Is `self` the path to a directory ?
```

Comment:
@privat `s/Is/Does/`
Or better IMHO: Is `self` a path to an existing directory?